arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/stack/Stack.java

```diff
@@ -26,7 +26,10 @@
 import org.hyperledger.besu.evm.frame.MessageFrame;
 
 public class Stack {
-  public static final int MAX_STACK_SIZE = 1024;
+  public static final short MAX_STACK_SIZE = 1024;
+  public static final byte NONE = 0;
+  public static final byte PUSH = 1;
+  public static final byte POP = 2;
```

Comment:
Looks good for now. But I would even go a step further and introduce these constants in besu.
ATM it's even hard to reason about what opcode `int` belongs to what. So if we ever need to change something it's easy to make a mistake: https://github.com/hyperledger/besu/blob/21e95a9efbdb95e7f70dfe5d89e76956c642c06e/evm/src/main/java/org/hyperledger/besu/evm/EVM.java#L231-L333

This can also be passed into operation constructors: https://github.com/hyperledger/besu/blob/21e95a9efbdb95e7f70dfe5d89e76956c642c06e/evm/src/main/java/org/hyperledger/besu/evm/operation/PopOperation.java#L33

I wouldn't be afraid to give it a static `int` as it will be inlined by the compiler.

Reply:
Yes, good catch, it should work for the evm as well. In the scope of this PR, the push opcode is more a general representation than a real opcode, as I guess it covers different push opcodes from 0x5f to 0x7f : https://www.evm.codes/?fork=cancun#5f

Reply:
In the evm, it is more for readability then performance I guess, maybe we can still replace int with byte. I remember also a discussion with Danno on this specific topic. He didn't want to store the opcodes numbers for some reason, especially when using Besu evm in another network, like Hedera.

Reply:
>He didn't want to store the opcodes numbers for some reason, especially when using Besu evm in another network, like Hedera.

I can't see why as well. Maybe because variants of PUSH make it awkward? But then again he is already doing it in a way in PUSH https://github.com/hyperledger/besu/blob/main/evm/src/main/java/org/hyperledger/besu/evm/operation/PushOperation.java#L27-L52.
Well not just for readability, now it would come handy if you want to use them and define them statically once.